projects/angular/src/modal/index.ts

```diff
@@ -8,3 +8,4 @@
 export * from './modal';
 export * from './modal.module';
 export * from './modal-body';
+export * from './sidebar';
```

Comment:
We discussed offline that it would be better to have the Sidebar in a separate module and not being part of the Modal.